docs/data-broker.apib

```diff
@@ -1,36 +1,33 @@
 FORMAT: 1A
 HOST: http://localhost:80
 
-# Dojot data broker API
+# Data Broker API
 
-This document describes the APIs using data-broker as a standalone process. If
-used within dojot, the endpoints might be changed a bit. Check [dojot's
-documentation](http://dojotdocs.readthedocs.io/) to check all of them.
+This document describes the APIs using Data Broker as a standalone process. If used within dojot,
+the endpoints might be changed a bit. Check
+[dojot's documentation](http://dojotdocs.readthedocs.io/) for more details.
 
 # Group Subject
 
-In order to allow clients to consume dojot events on their own pace, dojot
-exposes (both internally and externally) apache kafka streams. To manage the
-set of topics exposed by the kafka cluster, the data-broker
-component functions as a gatekeeper to the queues, configuring the needed
-streams on kafka and setting its accessibility parameters.
+In order to allow clients to consume dojot events on their own pace, dojot exposes (both internally
+and externally) Apache Kafka streams. To manage the set of topics exposed by the Kafka cluster, Data
+Broker functions as a gatekeeper to the queues, configuring the needed streams on Kafka and setting
```

Comment:
> [...] Data Broker **acts** as a gatekeeper [...]

I don't think you changed that, but if you can fix it, it's better to understand :+1: